packages/lighter-styleguide/src/scripts/esbuildReactDocgenTypescriptPlugin.ts

```diff
@@ -0,0 +1,95 @@
+import fs from "fs";
+import * as docgen from "react-docgen-typescript";
+
+// generationg docgenInfo is expansive. But there is no detection for file changes there for no invalidation is possible - docgenInfo will stay same during runtime
+const docgenInfoCache = new Map();
```

Comment:
@benistary toto imho nefunguje takto ako tu je urobne - tato cache - mozno by bolo treba napsiat aspon do komentara ze to nefunguje, ked to vieme? :)